specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/virtualNetworkGateway.json

```diff
@@ -447,6 +465,9 @@
             "description": "Accepted and the operation will complete asynchronously."
           }
         },
+        "x-ms-examples": {
+          "Generate Virtual Network Gateway VPN Profile": { "$ref": "./examples/VirtualNetworkGatewayGenerateVpnProfile.json" }
```

Comment:
By convention, json keys are named without spaces. Please fix in other instances too

Reply:
Removed spaces from example keys. Thanks for pointing that out.